### PR TITLE
ENH: indicate that on the Vidas you have to touch the screen to turn off the alarms

### PR DESCRIPTION
**Pull request by celprov**

### CODE REVIEW

File: docs/data-collection/notes-scanning.md [hunk 1]
```diff
@@ -162,6 +162,8 @@ Adjust settings by pressing the respective button and then turning the central k
 
 !!! warning "The central knob (button 1) will turn off the alarm if pushed when the alarm is on"
 
+!!! warning "On {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}, the alarm can be turned off by touching the tactil screen on the bore of the scanner"
```
Comment (oesteban) [inline docs/data-collection/notes-scanning.md]: Does the central knob work for this too? or is it just different?

Comment (celprov) [reply docs/data-collection/notes-scanning.md]: There is no central knob on the Vidas, the alarm can only be turned off by touching the screen

Comment (oesteban) [inline docs/data-collection/notes-scanning.md]: Okay, if that is the case, then we need to merge the two warnings and highlight that the procedure is different depending on the system:

```suggestion
!!! warning "Turning off the alarm"

    The alarm is turned off differently depending on the system:

       * {{settings.mri.models.mri1}}: push the central knob (button 1).
       * {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}: touch one of the tactile screens on the scanner's front.
```

File: docs/data-collection/notes-scanning.md [hunk 2]
```diff
@@ -160,7 +160,11 @@ Adjust settings by pressing the respective button and then turning the central k
 - [ ] Use the light :fontawesome-solid-lightbulb: button (4) to adjust the intensity of the illumination inside the scanning room.
 - [ ] Use the fan :fontawesome-solid-fan: button (5) to adjust the ventilation in the scanning room.
 
-!!! warning "The central knob (button 1) will turn off the alarm if pushed when the alarm is on"
+!!! warning "Turning off the alarm"
+
+    The alarm is turned off differently depending on the system:
+    * {{settings.mri.models.mri1}}, push the central knob (button 1). 
+    * {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}: touch one of the tactile screens on the scanner's front.
```
Comment (oesteban) [inline docs/data-collection/notes-scanning.md:167]: New lines are important here for rendering:

```suggestion
!!! warning "Turning off the alarm"

    The alarm is turned off differently depending on the system:

       * {{settings.mri.models.mri1}}: push the central knob (button 1).
       * {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}: touch one of the tactile screens on the scanner's front.
```

**Comment by oesteban** [comment]
@celprov - this is not rendering properly [notes](https://output.circle-artifacts.com/output/job/a9d8dc14-7c2f-48a7-8efd-9ae728bb8372/artifacts/0/_SOPs_preview/data-collection/notes-et/index.html)

Please check my comment https://github.com/TheAxonLab/hcph-sops/pull/497#discussion_r1731013028

**Pull request merged**